doc/rest-api.md

```diff
@@ -2424,3 +2427,236 @@ Input (none at present):
+ * Return: list of storage pools that are currently defined on the host
+
+    [
+        "/1.0/storage-pools/default",
+        "/1.0/storage-pools/pool1"
+        "/1.0/storage-pools/pool2"
+        "/1.0/storage-pools/pool3"
+        "/1.0/storage-pools/pool4"
+    ]
+
+### POST
+ * Description: create a new storage pool
+ * Introduced: with API extension `storage`
+ * Authentication: trusted
+ * Operation: sync
+ * Return: standard return value or standard error
+
+Input:
+
+    {
+        "config": {
+            "size": "10GB"
+        },
+        "driver": "zfs",
+        "name": "pool1"
+    }
+
+## `/1.0/cluster`
+### GET (optional `?password=<trust-password>`)
+ * Description: information about a cluster (such as networks and storage pools)
+ * Introduced: with API extension `clustering`
+ * Authentication: trusted or untrusted
+ * Operation: sync
+ * Return: dict representing a cluster
+
+    {
+        "type": "sync",
+        "status": "Success",
+        "status_code": 200,
+        "operation": "",
+        "error_code": 0,
+        "error": "",
+        "metadata": {
+            "storage_pools": [
+                {
+                    "name": "default",
+                    "description": "",
+                    "config": {
+                        "source": "/var/lib/lxd/storage-pools/default"
+                    },
+                    "driver": "dir",
+                    "used_by": null
+                }
+            ],
+            "networks": [
+                {
+                    "name": "lxdbr0",
+                    "description": "",
+                    "type": "bridge",
+                    "config": {
+                        "ipv4.address": "10.8.219.1/24",
+                        "ipv4.nat": "true",
+                        "ipv6.address": "fd42:f5a2:e47e:2185::1/64",
+                        "ipv6.nat": "true"
+                    },
+                    "used_by": null,
+                    "managed": true
+                }
+            ]
+    	}
+    }
+
+### DELETE
+ * Description: disable clustering
+ * Introduced: with API extension `clustering`
+ * Authentication: trusted
+ * Operation: sync
+ * Return: standard return value or standard error
+
+Input (none at present):
+
+    {
+    }
+
+## `/1.0/cluster/nodes`
+### GET
+ * Description: list of LXD nodes in the cluster
+ * Introduced: with API extension `clustering`
+ * Authentication: trusted
+ * Operation: sync
+ * Return: list of dicts with information about each node
+
+	{
+		"type": "sync",
+		"status": "Success",
+		"status_code": 200,
+		"operation": "",
+		"error_code": 0,
+		"error": "",
+		"metadata": [
+			{
+				"name": "lxd1",
+				"url": "https://10.1.1.101:8443",
+				"database": true,
+				"state": "ONLINE"
+			},
+			{
+				"name": "lxd2",
+				"url": "https://10.1.1.102:8443",
+				"database": true,
+				"state": "ONLINE"
+			},
+		]
+	} 
+
+### POST
+ * Description: bootstrap, join, or accept a node in the cluster
```

Comment:
Should be limited to adding (accepting) a node. Bootstrap and join should go through "POST /1.0/cluster" instead.

Reply:
Fixed as per the new API layout.